src/Either.ts

```diff
@@ -81,6 +81,18 @@ export class Left<L, A> {
   alt(fy: Either<L, A>): Either<L, A> {
     return fy
   }
+
+  /**
+   * Lazy version of {@link alt}
+   * @since 1.6.0
+   * @param {Lazy<Either<L, A>>} fy - thunk
+   * @example
+   * assert.deepEqual(right(1).altL(() => right(2)), right(1))
+   * @returns {Either<L, A>}
+   */
+  altL(fy: Lazy<Either<L, A>>): Either<L, A> {
```

Comment:
In case of `Either` I'd prefer a more general `orElse` method instead of `altL`

```ts
orElse<M>(f: (l: L) => Either<M, A>): Either<M, A> {
  ...
}
```

see `TaskEither.prototype.orElse`

Reply:
Should all `altL`s be renamed to `orElse`? Or only this because of `l: L` arg?

Reply:
Good idea, let's rename all `altL` to `orElse`